src/templates/admin/review/home.html

```diff
@@ -25,24 +25,37 @@ <h2>Articles in Review</h2>
                         <th>Editors</th>
                         <th>Section</th>
                         <th>Projected Issue</th>
-                        <th>No. Active Reviews</th>
+                        <th>Review Status</th>
```

Comment:
When we talked about this we said we would add a setting IIRC. I'm happy for it to be the default behaviour, but then we should probably remove the next column (No. Complete Reviews) since its a bit redundant.

Otherwise, add a setting and show either column depending on the setting value?